package.json

```diff
@@ -50,7 +51,7 @@
     "whatwg-fetch": "^3.6.2"
   },
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=20.18.0"
```

Comment:
Does the build really not work on Node versions before v20?

Reply:
It does, I changed this because of 2 things:
- Node 14,16, & 18 are deprecated.
- As we start to update and leverage our libraries more having a min version of 20.18.0 prevents us from having possible upgrade issues in the future. I'm eventually wanting to push us to React 19 mid next year.

TBH it's primarily helping with dependency management and preventing future headaches 😅.

Reply:
If Node's has already deprecated these versions, doesn't that make this unnecessary?